Tensor construction codemod(ResizeLike) - 3/3

jerryzh168 opened the pull request:
Summary:
Codemod generated with clangr shard mode, 25 files per diff,
motivation: https://github.com/pytorch/pytorch/pull/12407

Differential Revision: D13629082

smessmer reviewed (approved):
This diff has a lot of misplaced comments. Please fix them.

jerryzh168 committed:
Tensor construction codemod(ResizeLike) - 3/3 (#15943)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/15943

Codemod generated with clangr shard mode, 25 files per diff,
motivation: https://github.com/pytorch/pytorch/pull/12407

Reviewed By: smessmer

Differential Revision: D13629082

fbshipit-source-id: 8ffbb268b16f9d999f565a68c9755bb76c366d4d
